src/test/resources/pages/instructorCoursesAddDupIdFailed.html

```diff
@@ -112,7 +112,7 @@ <h2>
             Software Engineering $^&*()
           </td>
           <td data-date-stamp="${datetime.now.iso8601utc}" data-original-title="${datetime.now}" data-toggle="tooltip" id="coursecreateddate2">
-            ${datetime.now.courses}
+             ${datetime.now.courses}
```

Comment:
Unnecessary change like this (and in many other `*.html` files) should be reverted.